### PR TITLE
MO-59-Employees-change-term-Working-Hours-to-Budget

### PR DESCRIPTION
**Pull request by AndriiK139MCAF**
FINALY DONE IT !

### CODE REVIEW

File: mo_hr_timesheets/models/custome_contact_fields.py [hunk 1]
```diff
@@ -69,3 +69,14 @@ def _check_birthdays(self):
                         template_coworker_id.with_context(
                             birthday_employee=employee.name
                         ).send_mail(coworker.id, force_send=True)
+
+
+class CustomHrEmployeeBase(models.AbstractModel):
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/custome_contact_fields.py:74]: do we need abstract model here?

Comment (AndriiK139MCAF) [reply mo_hr_timesheets/models/custome_contact_fields.py:74]: Yes, we need it to give "String" value for this variable

Comment (MORPSmcaf) [reply mo_hr_timesheets/models/custome_contact_fields.py:74]: can you try 
(models.Model):
_inherit = 'hr.employee'

Comment (AndriiK139MCAF) [reply mo_hr_timesheets/models/custome_contact_fields.py:74]: Yes
but this is "AbstractModel"
so you need to use "AbstractModel"

**Pull request closed without merging**